app/views/admin/creator_settings/_form.html.erb

```diff
@@ -38,16 +38,20 @@
                          title: "Provide a valid HEX Color or pick your color from the color picker.",
                          placeholder: ::Settings::UserExperience.primary_brand_color_hex,
                          class: "crayons-textfield js-color-field",
-                         "data-action": "change->creator-settings#updateBranding" %>
+                         "data-action": "change->creator-settings#handleValidationsAndUpdates",
+                         "data-creator-settings-target": "brandColor" %>
       <%= color_field_tag :primary_brand_color_hex,
                           ::Settings::UserExperience.primary_brand_color_hex,
                           pattern: "^#+([a-fA-F0-9]{6})$",
                           placeholder: ::Settings::UserExperience.primary_brand_color_hex,
                           class: "crayons-color-selector js-color-field ml-2",
-                          "data-action": "change->creator-settings#updateBranding",
-                          required: true %>
+                          required: true,
+                          "data-action": "change->creator-settings#handleValidationsAndUpdates" %>
     </div>
   </div>
+  <div data-creator-settings-target="colorContrastError" class="mt-1 color-accent-danger hidden">
```

Comment:
We should make sure this error message is linked to the inputs it relates to by giving it an ID and adding to the inputs via `aria-describedby="id-of-error-message"`.

I think this will work best if we only populate the error message text when there is an error (like my above comment) since we don't want an error message linked with the input if the error isn't present in the form.

Reply:
✅